R/dummy_functions.R

```diff
@@ -1,81 +1,9 @@
-#' Get dummy `CDISC` data
-#'
-#' Get dummy `CDISC` data including `ADSL`, `ADAE` and `ADLB`.
-#' Some NAs are also introduced to stress test.
-#'
-#' @return `cdisc_data`
-#' @keywords internal
-example_cdisc_data <- function() { # nolint
-  ADSL <- data.frame( # nolint
-    STUDYID = "study",
-    USUBJID = 1:10,
-    SEX = sample(c("F", "M"), 10, replace = TRUE),
-    AGE = stats::rpois(10, 40)
-  )
-  ADTTE <- rbind(ADSL, ADSL, ADSL) # nolint
-  ADTTE$PARAMCD <- rep(c("OS", "EFS", "PFS"), each = 10) # nolint
-  ADTTE$AVAL <- c( # nolint
-    stats::rnorm(10, mean = 700, sd = 200), # dummy OS level
-    stats::rnorm(10, mean = 400, sd = 100), # dummy EFS level
-    stats::rnorm(10, mean = 450, sd = 200) # dummy PFS level
-  )
-
-  ADSL$logical_test <- sample(c(TRUE, FALSE, NA), size = nrow(ADSL), replace = TRUE) # nolint
-  ADSL$SEX[c(2, 5)] <- NA # nolint
-
-  res <- teal.data::cdisc_data(
-    ADSL = ADSL,
-    ADTTE = ADTTE,
-    code = '
-      ADSL <- data.frame(
-        STUDYID = "study",
-        USUBJID = 1:10,
-        SEX = sample(c("F", "M"), 10, replace = TRUE),
-        AGE = rpois(10, 40)
-      )
-      ADTTE <- rbind(ADSL, ADSL, ADSL)
-      ADTTE$PARAMCD <- rep(c("OS", "EFS", "PFS"), each = 10)
-      ADTTE$AVAL <- c(
-        rnorm(10, mean = 700, sd = 200),
-        rnorm(10, mean = 400, sd = 100),
-        rnorm(10, mean = 450, sd = 200)
-      )
-
-      ADSL$logical_test <- sample(c(TRUE, FALSE, NA), size = nrow(ADSL), replace = TRUE)
-      ADSL$SEX[c(2, 5)] <- NA
-    '
-  )
-  return(res)
-}
-
-#' Get datasets to go with example modules.
-#'
-#' Creates a nested list, the structure of which matches the module hierarchy created by `example_modules`.
-#' Each list leaf is the same `FilteredData` object.
-#'
-#' @return named list of `FilteredData` objects, each with `ADSL` set.
-#' @keywords internal
-example_datasets <- function() { # nolint
-  dummy_cdisc_data <- example_cdisc_data()
-  datasets <- teal_data_to_filtered_data(dummy_cdisc_data)
-  list(
-    "d2" = list(
-      "d3" = list(
-        "aaa1" = datasets,
-        "aaa2" = datasets,
-        "aaa3" = datasets
-      ),
-      "bbb" = datasets
-    ),
-    "ccc" = datasets
-  )
-}
-
 #' An example `teal` module
 #'
-#' @description `r lifecycle::badge("experimental")`
+#' `r lifecycle::badge("experimental")`
+#'
 #' @inheritParams module
-#' @return A `teal` module which can be included in the `modules` argument to [teal::init()].
+#' @return A `teal` module which can be included in the `modules` argument to [`teal::init()`].
```

Comment:
According to [roxygen2 instruction](https://roxygen2.r-lib.org/articles/rd-formatting.html#links), there is no tag which has backtick around fun `[´fun()´]`. Backtick suggest only topics

Reply:
reverted  changes for backticks [a6b98fe](https://github.com/insightsengineering/teal/pull/1021/commits/a6b98fe60ad5a73cd3cdba3580c9576b6186d3b1)